core/src/main/resources/resources/ModuleManager_de.properties

```diff
@@ -1,6 +1,6 @@
 serverSettings.manageModules=Module verwalten
-serverSettings.manageModules.availableModules=Verf�gbare Module
-serverSettings.manageModules.checkForUpdates=Auf Updates pr�fen
+serverSettings.manageModules.availableModules=Verf�gbare Module
```

Comment:
Be careful with encoding, you must use iso-8859-1

Reply:
Make sense, I'll change it

Reply:
changed